src/app/components/data-preview/data-preview.component.ts

```diff
@@ -63,11 +63,11 @@ export class DataPreviewComponent implements OnChanges {
                 dataType: this.data.type,
                 contentType: this.data.contentType,
             };
-        } else if (isStepApiObject(this.data)) {
+        } else if (isStepLikeApiObject(this.data)) {
             previewData = {
-                url: this.backend.backendRootUrl + this.data.parameters,
+                url: this.backend.backendRootUrl + (this.data.parameters ?? ""),
```

Comment:
Preview data should be null if the required attributes are not available. This poses the risk of previewing a more or less random api enpoint...